agent/command/git.go

```diff
@@ -454,6 +465,10 @@ func (c *gitFetchProject) Execute(ctx context.Context, comm client.Communicator,
 		func() (bool, error) {
 			err := c.fetch(ctx, comm, logger, conf, opts)
 			if err != nil {
+				if opts.cloneParams != "" && strings.Contains(err.Error(), "exit status 129") {
```

Comment:
what is this exist status 129? can we make this a function constant as opposed to hardcoding here, with a comment?

Reply:
removed